Tasks/Common/azure-arm-rest/package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "q": "1.4.1",
     "vsts-task-lib": "2.0.5",
-    "typed-rest-client": "0.12.0"
+    "typed-rest-client": "0.12.0",
+    "xml2js": "0.4.13"
```

Comment:
Does this change the TPN for other tasks that take dependency on this common module?

Reply:
AzureKeyVault seems to have TPN.txt
@kmkumaran ,Should I loop in PM to get the above file or copying it from our task is acceptable?